onnxscript/converter.py

```diff
@@ -1352,9 +1361,15 @@ def translate_function_def(self, fn: ast.FunctionDef) -> irbuilder.IRFunction:
                 self.returntype = None
         else:
             self.returntype = None
+
+        return self._current_fn
+
+    def translate_function_def(self, fn: ast.FunctionDef) -> irbuilder.IRFunction:
```

Comment:
This function now looks awkwardly short. Maybe leave function descriptions on `translate_function_signature`, or comment here?

Reply:
I added a docstring. I think its quite natural to handle the signature and the body in two steps like we do now?